hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/CommitsCommand.java

```diff
@@ -131,8 +133,13 @@ public String rollbackCommit(@CliOption(key = {"commit"}, help = "Commit to roll
     }
 
     SparkLauncher sparkLauncher = SparkUtil.initLauncher(sparkPropertiesPath);
-    sparkLauncher
-        .addAppArgs(SparkMain.SparkCommand.ROLLBACK.toString(), commitTime, HoodieCLI.tableMetadata.getBasePath());
+
+    HoodieCommitRollbackJobConfig config = new HoodieCommitRollbackJobConfig();
```

Comment:
@bvaradar thanks for reviewing the changes. I am working on the rest of the changes. I got the reasoning that a hoodie-common module is a place of core classes. It would be good if we rename this module as hoodie-core instead of hoodie-common. 

Also, I have a question regarding this comment. Do you want me to put all the command configs classes into their perspective original classes/package(where it was before) or in a hoodie-client module?

Reply:
@abhioncbr I was suggesting to put all config classes back as static nested in their original place.

Reply:
All the command configs classes are moved back to their original place.